src/overlays/actors/ovl_En_Invadepoh/z_en_invadepoh.c

```diff
@@ -3873,8 +3872,8 @@ void func_80B4C3A0(Actor* thisx, PlayState* play) {
         this->actor.objBankIndex = this->bankIndex;
         Actor_SetObjectDependency(play, &this->actor);
         func_80B44FEC();
-        SkelAnime_InitFlex(play, &this->skelAnime, &object_ma2_Skel_015C28, &object_ma2_Anim_016720, this->jointTable,
-                           this->morphTable, 22);
+        SkelAnime_InitFlex(play, &this->skelAnime, &gCremiaSkel, &gCremiaWalkAnim, this->jointTable, this->morphTable,
+                           22);
```

Comment:
```suggestion
                           CREMIA_LIMB_MAX);
```

Reply:
Thanks, I consistently forget to do this in Invadepoh for some reason.